### PR TITLE
[SPARK-35588][PYTHON][DOCS] Update quickstart.ipynb to use pyspark.pandas

### PR DESCRIPTION
**Pull request by yos1p**
### What changes were proposed in this pull request?
Minor update to the notebook to show how to use pyspark.pandas package. Also fix the URL to directly go to the latest pyspark documentation.

### Why are the changes needed?
Fix documentation for PySpark quickstart.

### Does this PR introduce _any_ user-facing change?
No

### How was this patch tested?
All cells run with expected output, without any error.

### CODE REVIEW

**Comment by holdenk** [comment]
Jenkins OK to test

**Comment by HyukjinKwon** [comment]
Thanks for the contribution @yos1p. There are actually notebooks dedicated for pandas APIs on Spark: https://github.com/apache/spark/tree/master/python/docs/source/getting_started and we should merge or place it appropriately in the docs.

**Comment by HyukjinKwon** [comment]
@yos1p would you mind enabling GA builds? Apache Spark development uses the resources in your forked repository. See https://github.com/apache/spark/pull/32795/checks?check_run_id=2760628549

**Comment by yos1p** [comment]
> Thanks for the contribution @yos1p. There are actually notebooks dedicated for pandas APIs on Spark: https://github.com/apache/spark/blob/master/python/docs/source/getting_started/ps_10mins.ipynb and we should merge or place it appropriately in the docs.

Because most of the code in quickstart focuses on working with Spark DataFrame, I decided to put the spark pandas quick start in a new section instead of merging in the sections specialized for Spark DataFrame. Let me know what you think.

**Comment by HyukjinKwon** [comment]
Also please remove https://github.com/apache/spark/blob/master/python/docs/source/getting_started/ps_10mins.ipynb (assuming that your proposal merge both).

**Comment by HyukjinKwon** [comment]
I don't have a concrete idea for now. I will take a look and see if it makes sense.

File: python/docs/source/getting_started/ps_install.rst [hunk 1]
```diff
@@ -1,145 +0,0 @@
-============
```
Comment (HyukjinKwon) [inline python/docs/source/getting_started/ps_install.rst:1]: I think we can keep this and ps_videos_blogs.rst for now. They will be either updated, removed or merged at SPARK-35645

Comment (yos1p) [reply python/docs/source/getting_started/ps_install.rst]: Okay, restored.

**Comment by HyukjinKwon** [comment]
@yos1p, can you make sure Binder works correctly? Your PR can be tested here: https://mybinder.org/v2/gh/yos1p/spark/spark-35588?filepath=python%2Fdocs%2Fsource%2Fgetting_started%2Fquickstart.ipynb

You might need to push some changes at https://github.com/apache/spark/blob/master/binder/postBuild to build Spark, install extra dependencies, etc.

**Comment by yos1p** [comment]
> You might need to push some changes at https://github.com/apache/spark/blob/master/binder/postBuild to build Spark, install extra dependencies, etc.

It seems mybinder not able to include build spark. I always get error 255, which I believe caused by the building process taking too long. Any idea how to solve this? @HyukjinKwon

**Pull request closed without merging**